src/input_xml.F90

```diff
@@ -2417,73 +2422,6 @@ subroutine read_materials_xml(libraries, library_dict, material_temps)
         end do INDIVIDUAL_NUCLIDES
       end if
 
-      ! =======================================================================
-      ! READ AND PARSE <element> TAGS
-
-      ! Get pointer list of XML <element>
-      call get_node_list(node_mat, "element", node_ele_list)
```

Comment:
Instead of completely ignoring `<element>`, we ought to abort on inputs that still have `<element>`s within a material.